modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/spaces_navigation/SpacesNavigation.tsx

```diff
@@ -10,26 +10,26 @@ import ClayPanel from '@clayui/panel';
 import {sub} from 'frontend-js-web';
 import React, {useEffect, useState} from 'react';
 
-import {getAssetsLibrariesByCompany} from '../../api/api';
+import SpaceService from '../../structure_builder/services/SpaceService';
```

Comment:
This import of SpaceService is not needed here, is it?

Reply:
You're right. Fixed in the "Auto SF" commit.